app/controllers/ontologies_metadata_curator_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OntologiesMetadataCuratorController < ApplicationController
+    layout :determine_layout
+
+
+
+    def result
+        @ontologies_ids = params[:ontology][:ontologyId].drop(1)
+        @metadata_sel = params[:search][:metadata].drop(1)
+        @ontologies = []
+        @submissions = []
+        @ontologies_ids.each do |data|
+            @ontologies << LinkedData::Client::Models::Ontology.all.detect {|e| e.acronym == data}
+            @submissions << LinkedData::Client::Models::OntologySubmission.all.detect {|e| e.ontology.acronym == data}
+        end
+        if @ontologies.empty?
+            @ontologies = LinkedData::Client::Models::Ontology.all.sort_by {|e| e.name}
+            @submissions = LinkedData::Client::Models::OntologySubmission.all.sort_by {|e| e.ontology.name}
+        end
+        respond_to do |format|
+            format.html { redirect_to admin_index_path}
+            format.turbo_stream { render turbo_stream: turbo_stream.append("choices", partial: "ontologies_metadata_curator/result") }
```

Comment:
replace the frame name "choices", with something more specific that has a meaning.